go.mod

```diff
@@ -5,31 +5,31 @@ go 1.21
 require (
 	github.com/MakeNowJust/heredoc v1.0.0
 	github.com/google/btree v1.1.2
-	github.com/google/go-cmp v0.5.9
+	github.com/google/go-cmp v0.6.0
 	github.com/google/gofuzz v1.2.0
 	github.com/google/uuid v1.6.0
```

Comment:
kube is on `v1.3.0` - https://github.com/kubernetes/kubernetes/blob/release-1.29/go.mod#L47

this dep was changed earlier, I think it is okay since minor version are compatible.